core/controllers/custom_landing_pages.py

```diff
@@ -38,7 +38,8 @@ def get(self, subject, topic):
         """Handles GET requests."""
         if subject in feconf.AVAILABLE_LANDING_PAGES:
             if topic in feconf.AVAILABLE_LANDING_PAGES[subject]:
-                self.render_template('pages/landing/topic_landing_page.html')
+                (self.render_template(
```

Comment:
You don't need the outer parens. Here and elsewhere.

Reply:
If i don't do this then pylint will throws an error of Long-line.
Fot that reason only i have do it.

Reply:
done